pkg/types/config_test.go

```diff
@@ -18,27 +18,28 @@ package types
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestBuildUrl(t *testing.T) {
 	cfgNoPath := SecretConfig{Host: "localhost", Port: 8080, Protocol: "http"}
-	cfgWithPath := SecretConfig{Host: "localhost", Port: 8080, Protocol: "http", Path: "/ping"}
-	cfgWithTrailingSlash := SecretConfig{Host: "localhost", Port: 8080, Protocol: "http", Path: "/api/v1/ping/"}
-	cfgWithNoHost := SecretConfig{Host: "", Port: 8080, Protocol: "http", Path: ""}
-	cfgWithInvalidHost := SecretConfig{Host: "not valid", Port: 8080, Protocol: "http", Path: ""}
-	cfgWithUnsetPort := SecretConfig{Host: "", Port: 0, Protocol: "http", Path: "/api/v1/ping/"}
-	cfgWithInvalidPort := SecretConfig{Host: "", Port: 9999, Protocol: "http", Path: "/api/v1/ping/"}
-	cfgWithNoProtocol := SecretConfig{Host: "localhost", Port: 8080, Protocol: "", Path: "/api/v1/ping/"}
-	cfgWithInvalidProtocol := SecretConfig{Host: "localhost", Port: 8080, Protocol: "234", Path: ""}
+	cfgWithPath := SecretConfig{Host: "localhost", Port: 8080, Protocol: "http", StoreBasePath: "/ping"}
```

Comment:
Humm, now that we see that SecretConfig is used for both Secrets and general Vault requests, not sure `StoreBasePath` is the correct name. Maybe just `basePath`